_source/_posts/2019-05-31-simple-auth-express-fifteen-minutes.md

```diff
@@ -183,7 +214,7 @@ npm install -g uuid-cli
 echo "APP_SECRET=`uuid`" >> .env
 ```
 
-Next, log in to your Okta developer console, navigate to **Applications**, then click **Add Application**. Select **Web**, then click **Next**. Change all ports to be 3000 and add `http://localhost:3000/callback` as a **Login redirect URI**.
+Next, log in to your Okta developer console, navigate to **Applications** > **Applications**, then click **Create App Integration**. Select **OIDC** under sign-in method and then select **Web Application** under application type, then click **Next**. Change all ports to be 3000 and add `http://localhost:3000/callback` as a **Sigh-in redirect URI**. Under assignments, select **Allow everyone in your organization to access** and then click **Save**.
```

Comment:
When we updated hundreds of blog posts in March of this year, we tried to change them to use the CLI instead of the Admin Console. There's a how to guide at https://developer.okta.com/blog/setup. 

The reason we changed to this is 1) we don't want to have screencasts that get outdated again in the future and 2) we should be able to link to our docs for the Admin Console setup, which the CLI include has in it.

Reply:
so should we change to use CLI here as well

Reply:
done